views/logs.pug

```diff
@@ -51,6 +48,4 @@ block content
       value: new Date(),
       showTimezone: true,
       timezone: "UTC"
-    });
-
-
+    });
```

Comment:
Only because GitHub shows a big warning symbol: there is no newline at the end of the file